server/src/main/java/org/elasticsearch/index/reindex/AbstractBulkByScrollRequest.java

```diff
@@ -130,7 +130,6 @@ public AbstractBulkByScrollRequest(SearchRequest searchRequest, boolean setDefau
         if (setDefaults) {
             searchRequest.scroll(DEFAULT_SCROLL_TIMEOUT);
             searchRequest.source(new SearchSourceBuilder());
-            searchRequest.source().size(DEFAULT_SCROLL_SIZE);
```

Comment:
I *think* this should stay as it was or else transport client users will start to get a default scroll size of `10` again which won't perform well..